root/static/scripts/edit/forms.js

```diff
@@ -29,7 +29,9 @@ MB.forms = {
         var nbsp = String.fromCharCode(160);
 
         function buildOptions(parent, indent) {
-            var i = 0, children = parent.children, child;
+            let i = 0;
+            const children = parent.children;
+            let child;
             if (!children) { return; }
```

Comment:
There's now a conflict here with the master branch which needs resolving, though when you fix it, I notice we don't need to declare `child` or `i` until after the `return`, so I'd do:
```JS
            const children = parent.children;

            if (!children) {
                return;
            }

            let child;
            let i = 0;
```